### PR TITLE
🌱 Adding Github Action to verify title

### PR DESCRIPTION
**Pull request by parthdhanjal**
Description

Add Github Action to verify the title so that we can generate the release notes

Closes: https://github.com/operator-framework/audit/issues/133

### CODE REVIEW

File: .github/workflows/TitleVerifier.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+  runs-on: ubuntu-latest
+  verify:
+    name: Verify PR Title
+    steps:
+    - name: Verify Title
+      id: verifier
+      uses: kubernetes-sigs/kubebuilder-release-tools@v0.2.0
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (camilamacedo86) [inline .github/workflows/TitleVerifier.yaml:17]: Could give an space at the end of this file?

File: .github/workflows/TitleVerifier.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Title Verifier
+
+on:
+  pull_request_target:
+    types: [push, opened, edited, reopened]
+
```
Comment (camilamacedo86) [inline .github/workflows/TitleVerifier.yaml:6]: Could you please rename the file for we do not have capitalized letters? 
For example, could it be title-verify.yml

**Review by gallettilance — CHANGES_REQUESTED** [review]
Please squash the commits

File: .github/workflows/TitleVerifier.yaml (hunk 1 above)
Comment (camilamacedo86) [reply .github/workflows/TitleVerifier.yaml]: done

File: .github/workflows/TitleVerifier.yaml (hunk 2 above)
Comment (camilamacedo86) [reply .github/workflows/TitleVerifier.yaml]: done

**Pull request merged**